### DISCUSSION TITLE
How to change the size of a slider?

### DISCUSSION THREAD

**Discussion by galoomba1** [emilk/egui#321] [Q&A] [answered]
I have a slider that i want to be able to move precisely, so i want it to be long (on the screen).

**Comment by emilk** [comment]
You can use https://docs.rs/egui/0.11.0/egui/struct.Ui.html#method.add_sized or https://docs.rs/egui/0.11.0/egui/style/struct.Spacing.html#structfield.slider_width

**Reply by galoomba1** [reply]
> 
> 
> You can use https://docs.rs/egui/0.11.0/egui/struct.Ui.html#method.add_sized or https://docs.rs/egui/0.11.0/egui/style/struct.Spacing.html#structfield.slider_width

The first way just doesn't work; the second way i have no idea how to use, nothing i try works

**Reply by emilk** [reply]
Ah my mistake, `add_sized` does indeed not work. But `ui.spacing_mut().slider_width = 1000.0; ui.add(Slider::new(…));` should work fine

**Reply by galoomba1** [reply]
Found that already and it works 👍

**Comment by galoomba1** [comment, marked as answer]
This works https://github.com/emilk/egui/discussions/177#discussioncomment-382295
